src/boot/bootctl.c

```diff
@@ -1465,6 +1467,8 @@ static int help(int argc, char *argv[], void *userdata) {
                "                       Generate JSON output\n"
                "     --all-architectures\n"
                "                       Install all supported EFI architectures\n"
+               "     --efi-label=STRING\n"
```

Comment:
Not a matter, but maybe `STRING` -> `NAME` or `LABEL` is better?

Reply:
Done.